node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs');
+
+fs.readFile(`/workspaces/rc0522-code-solutions/node-fs-readfile/${process.argv[2]}`, 'utf8', (err, data) => {
```

Comment:
Here we want to use a relative path, using an absolute path will only work on your system. If I ran this code it would break, so we want to use relative paths to make it more universal.
```JAVASCRIPT
fs.readFile(process.argv[2], 'utf8', (err, data) => {
```